codespell_lib/_codespell.py

```diff
@@ -389,11 +444,14 @@ def parse_options(args):
     parser.add_argument('-C', '--context', type=int, metavar='LINES',
                         help='print LINES of surrounding context')
     parser.add_argument('--config', type=str,
-                        help='path to config file.')
+                        help='path to config file.',
+                        ).complete = shtab.FILE
     parser.add_argument('--toml', type=str,
-                        help='path to a pyproject.toml file.')
+                        help='path to a pyproject.toml file.',
+                        ).complete = TOML_FILE
```

Comment:
I know this wasn't entirely you, but if it has to be a pyproject.toml file, rather than just a .toml file, we should restrict the completion, if it doesn't, maybe the help should say:
```suggestion
                        help='path to a pyproject.toml format file.',
                        ).complete = TOML_FILE
```

or:
```suggestion
                        help='path to a pyproject.toml type file.',
                        ).complete = TOML_FILE
```

Reply:
I'll fix it.